features/device-mgt/org.wso2.carbon.device.mgt.server.feature/src/main/resources/dbscripts/cdm/mysql.sql

```diff
@@ -585,3 +585,16 @@ ORDER BY TENANT_ID, DEVICE_ID;
 
```

Comment:
This query is removed from this file and added to carbon-device-mgt/features/certificate-mgt/org.wso2.carbon.certificate.mgt.server.feature/src/main/resources/dbscripts/certMgt/mysql.sql and formatting has done.